parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -111,6 +111,8 @@ function basePrep {
         logs_to_events "AKS.CSE.removeManDbAutoUpdateFlagFile" removeManDbAutoUpdateFlagFile
     fi
 
+    logs_to_events "AKS.CSE.getIMDSMetadataBody" get_imds_instance_metadata
```

Comment:
Do you need it now? I guess it can be useful for metrics collections.
I don't have a strong opinion here.